main.tf

```diff
@@ -14,11 +14,28 @@
  * limitations under the License.
  */
 
-terraform {
```

Comment:
This configuration should not be dropped; it enforces the supported version.

Reply:
Bringing it back